BATCH_MANUAL.md

```diff
@@ -0,0 +1,98 @@
+```leaks_output_{simulation}_{Program name}.csv```
+```sites_output_{simulation}_{Program name}.csv```
+
+### Sites
+
+`sites_output_{program}_concat.csv`
+
+|Column Names      |Unit   |Description                |
+|------------------|-------|---------------------------|
+|facility_ID       |N/A    |Facility ID of the site|
+|Cum_leaks         |N/A    |Average cumulative number of leaks at the site|
+|init_leaks        |N/A    |Average number of initial leaks found at the site|
+|total_emissions_kg|kg     |Average total emissions per site (in kilograms)|
+|subtype_code      |N/A    |The subtype code of the site|
+|lat               |Degrees|Latitude of the site|
+|lon               |Degrees|Longitude of the site|
+|equipment_groups  |N/A    |Number of equipment groups at the site|
+|n_sims            |N/A    |Number of simulations the site participated in|
+
+```sites_summary.csv```
+
+For this file, each individual row is representing the summary statistics of a single program of a single simulation run.
+
+|Column Names                       |Unit       |Description                |
+|-----------------------------------|-----------|---------------------------|
+|Program                            | N/A       |Name of the program             |
+|Mean_emissions_per_site            |per site   |Mean emissions per site in kg for a single site, calculated for each simulation set|
+|5th_percentile_Emissions_per_site  |per site   |5th percentile of emissions per site in kilograms, calculated for each simulation set|
+|95th_percentile_Emissions_per_site |per site   |95th percentile of emissions per site in kilograms, calculated across for each simulation set|
+|Mean_leaks_per_site                | # per site|Mean number of leaks per site, calculated across all simulation sets|
+|5th_percentile_leaks_per_site      | # per site|5th percentile of the number of leaks per site at a given site, calculated for each simulation set|
+|95th_percentile_leaks_per_site     | # per site|95th percentile of the number of leaks per site at a given , calculated for each simulation set|
+
+### Leaks
+
+```leaks_summary.csv```
+
+For this file, each individual row is representing the summary statistics of a single program of a single simulation run.
+
+|Column Names                |Unit   |Description                |
+|----------------------------|-------|---------------------------|
+|Program                     | N/A   |Name of the program|
+|Volume_mean                 |kg     |Mean emissions volume in kilograms of leaks, calculated for a run of the simulation|
+|5th_percentile_Volume       |kg     |5th percentile emissions volume in kilograms of leaks, calculated for a run of the simulation|
+|95th_percentile_Volume      |kg     |95th percentile emissions volume in kilograms of leaks, calculated for a run of the simulation|
+|Mean_leak_rate              |g/sec  |Overall mean leak rate for all leaks in a simulation|
```

Comment:
This is really per leak, calculated for a run in the simulation for all of these stats